x-pack/plugins/data_visualizer/public/application/data_comparison/data_comparison_overview_table.tsx

```diff
@@ -0,0 +1,290 @@
+import { i18n } from '@kbn/i18n';
+import {
+  EuiBasicTableColumn,
+  EuiButtonIcon,
+  EuiIcon,
+  EuiInMemoryTable,
+  EuiScreenReaderOnly,
+  EuiTableFieldDataColumnType,
+  EuiToolTip,
+} from '@elastic/eui';
+import { COLLAPSE_ROW, EXPAND_ROW } from '../../../common/i18n_constants';
+import { COMPARISON_LABEL, DATA_COMPARISON_TYPE_LABEL, REFERENCE_LABEL } from './constants';
+import { useCurrentEuiTheme } from '../common/hooks/use_current_eui_theme';
+import { DataComparisonField, Feature, FETCH_STATUS } from './types';
+import { formatSignificanceLevel } from './data_comparison_utils';
+import { SingleDistributionChart } from './charts/single_distribution_chart';
+import { OverlapDistributionComparison } from './charts/overlap_distribution_chart';
+import { DataComparisonDistributionChart } from './charts/data_comparison_distribution_chart';
+
+const dataComparisonYesLabel = i18n.translate(
+  'xpack.dataVisualizer.dataComparison.fieldTypeYesLabel',
+  {
+    defaultMessage: 'Yes',
+  }
+);
+const dataComparisonNoLabel = i18n.translate(
+  'xpack.dataVisualizer.dataComparison.driftDetectedNoLabel',
+  {
+    defaultMessage: 'No',
+  }
+);
+
+export const DataComparisonOverviewTable = ({
+  data,
+  onTableChange,
+  pagination,
+  sorting,
+  status,
+}: {
+  data: Feature[];
+  status: FETCH_STATUS;
+} & UseTableState<Feature>) => {
+  const euiTheme = useCurrentEuiTheme();
+  const colors = {
+    referenceColor: euiTheme.euiColorVis2,
+    productionColor: euiTheme.euiColorVis1,
+  };
+  const [itemIdToExpandedRowMap, setItemIdToExpandedRowMap] = useState<Record<string, ReactNode>>(
+    {}
+  );
+
+  const referenceDistributionLabel = i18n.translate(
+    'xpack.dataVisualizer.dataComparison.dataComparisonDistributionLabel',
+    {
+      defaultMessage: '{label} distribution',
+      values: { label: REFERENCE_LABEL },
+    }
+  );
+  const comparisonDistributionLabel = i18n.translate(
+    'xpack.dataVisualizer.dataComparison.dataComparisonDistributionLabel',
+    {
+      defaultMessage: '{label} distribution',
+      values: { label: COMPARISON_LABEL },
+    }
+  );
+
+  const columns: Array<EuiBasicTableColumn<Feature>> = [
+    {
+      align: 'left',
+      width: '40px',
+      isExpander: true,
+      name: (
+        <EuiScreenReaderOnly>
+          <span>{EXPAND_ROW}</span>
+        </EuiScreenReaderOnly>
+      ),
+      render: (item: Feature) => {
+        const itemIdToExpandedRowMapValues = { ...itemIdToExpandedRowMap };
+
+        return (
+          <EuiButtonIcon
+            onClick={() => toggleDetails(item)}
+            aria-label={itemIdToExpandedRowMapValues[item.featureName] ? COLLAPSE_ROW : EXPAND_ROW}
+            iconType={itemIdToExpandedRowMapValues[item.featureName] ? 'arrowDown' : 'arrowRight'}
+          />
+        );
+      },
+    },
+
+    {
+      field: 'featureName',
+      name: i18n.translate('xpack.dataVisualizer.dataComparison.fieldNameLabel', {
+        defaultMessage: 'Field name',
```

Comment:
In the Data Visualizer we use `Name` as the column heading. We should be consistent across the views. Also noted that in the Log rate analysis table we use `Field name`.